explorer/src/components/account/StakeAccountSection.tsx

```diff
@@ -114,12 +139,17 @@ function OverviewCard({
             </td>
           </tr>
         )}
-        {!stakeAccount.meta && (
-          <tr>
-            <td>State</td>
-            <td className="text-lg-right">{TYPE_NAMES[stakeAccountType]}</td>
-          </tr>
-        )}
+        {!stakeAccount.meta ||
```

Comment:
While you're in the area could you clean up the conditionals with `stakeAccount.meta`? It will always be set